tests/unit/utils/logger.test.ts

```diff
@@ -0,0 +1,61 @@
+import { stub } from 'sinon'
+import { afterEach, test, expect } from 'vitest'
```

Comment:
```suggestion
import { afterEach, test, expect, vi } from 'vitest'
```

Then we can again remove sinon.